packages/keyring-controller/package.json

```diff
@@ -31,7 +31,7 @@
   "dependencies": {
     "@keystonehq/metamask-airgapped-keyring": "^0.13.1",
     "@metamask/base-controller": "^3.2.3",
-    "@metamask/eth-keyring-controller": "^13.0.1",
+    "@metamask/eth-keyring-controller": "^14.0.0",
```

Comment:
Just checking the dependency tree. Based on the packages being updated downstream, this does not seem like it would actually introduce breaking changes. The updates seem to mainly be to unrelated parts of `@metamask/utils`. The bump to `ethereum-cryptography` _could_ be breaking but doesn't seem that way.

Would you agree?

Reply:
Could it exacerbate the issue here? https://github.com/MetaMask/core/pull/1868